<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>618</width>
    <height>443</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RemotePyLauncher</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>481</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Welcome to the RemotePyLauncher.&lt;br&gt;Version is xxxxx</string>
    </property>
   </widget>
   <widget class="QWidget" name="formLayoutWidget">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>50</y>
      <width>421</width>
      <height>319</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <item row="4" column="0">
      <widget class="QLabel" name="scriptDependenciesLabel">
       <property name="text">
        <string>Script Dependencies</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="scriptDependenciesLineEdit"/>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="outputFormatLabel">
       <property name="text">
        <string>Output format</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QComboBox" name="outputFormatComboBox"/>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="outputFilenameLabel">
       <property name="text">
        <string>Output filename</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QLineEdit" name="outputFilenameLineEdit"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="chooseFileToLaunchLabel">
       <property name="text">
        <string>Script to run</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="chooseFileToLaunchLineEdit">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="readOnly">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="scriptTypeLabel">
       <property name="text">
        <string>Script Type</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QComboBox" name="scriptTypeComboBox"/>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="savedOutputsDirectoryLabel">
       <property name="text">
        <string>Save Output As</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QLineEdit" name="savedOutputsDirectoryLineEdit">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QCheckBox" name="SaveStreamsCheckbox">
       <property name="text">
        <string>Also save standard streams
(stdout, stderr) to the same location</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="serverLabel">
       <property name="text">
        <string>Server Endpoint</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="serverLineEdit">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="serverTokenLabel">
       <property name="text">
        <string>Server Token</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="serverTokenLineEdit"/>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>380</y>
      <width>121</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Send to Server</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>120</y>
      <width>83</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Choose</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>290</y>
      <width>83</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Choose</string>
    </property>
   </widget>
   <widget class="QLabel" name="loadingLabel">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>350</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>loadingLabel</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
